references/data-converters-shared-flow/pipeline.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+./deploy.sh
```

Comment:
This pipeline only deploys the sf but doesn't execute the transformation logic. In this way it mainly tests sackmesser. 

In an ideal world we'd have a test proxy that uses the shared flow and something like an apickli test to validate the result.

The test proxy should probably contain the following endpoints

/mock-bq
/transform-bq
/mock-odata
/transform-odata
/mock-firestore
/transform-firestore

Reply:
also maybe add the js unit tests to the pipeline?

Reply:
would love to do that, but would also take some time to get setup.. can I promise it for the next version, and for now just add the unit tests to the pipeline?

Reply:
Fine with me. the Unit test covers the main logic sufficiently for a first stab.